start_your_career/start_your_career.html

```diff
@@ -85,70 +47,67 @@ <h2 style="text-align:center;">
       Vel incidunt dolorum obcaecati, hic rem quaerat quasi quo officia esse voluptate, autem, nostrum aliquid accusantium? Tempora veritatis impedit, vel tenetur sapiente repudiandae et culpa facilis? Repellat, aspernatur natus! Dicta.
       Odio commodi eum repellendus dolores velit delectus, esse voluptatibus quam impedit magni quisquam maiores quidem voluptatem minima. At cupiditate sit animi asperiores, eum ipsum enim doloribus provident quam corporis mollitia!
     </div>
-
-    <h2 class="headers">DEMANDED PROGRAMS</h2>
+    <h2>DEMANDED PROGRAMS</h2>
     <hr>
-    <div class="block">Programming</div>
-    <div class="block">Testing</div>
-    <div class="block">Business analysis</div>
-    <div class="block" >Trainig courses</div>
-    <div class="block">Web design and computer graphics</div>
-    <div class="block">Marketing & Sales</div>
-    <div class="block">System and network administration</div>
-    <div style="clear: both;"></div>
-
-    <h2 style="text-align:center; margin: 25px;">UPCOMING COURSES</h2>
+         <div class="block">Programming</div>
+         <div class="block">Testing</div>
+         <div class="block">Business analysis</div>
+         <div class="block">Trainig courses</div>
+         <div class="block">Web design and computer graphics</div>
+         <div class="block">Marketing & Sales</div>
+         <div class="block">System and network administration</div>
+         <div class="clear"></div>
+    <h2>UPCOMING COURSES</h2>
     <hr>
-    <table style="border-collapse: collapse; border-radius: 5px; margin-top: 10px; margin-bottom: 10px; 
-                  width: 100%; background-color: white; table-layout: fixed;">
-   <tr style="border-bottom: 1px grey solid;">
+    <div class="table"> 
```

Comment:
class="table" and the again class="table"?)

Reply:
The first div with class "table" is necessary for the adaptive table.